.github/CODEOWNERS

```diff
@@ -619,6 +619,9 @@
 # ServiceLabel: %Mobile Engagement
 # ServiceOwners:                                                   @kpiteira
 
+# PRLabel: %Mobile Network
```

Comment:
As this is a management package, please move to the management section around L947 to preserve alphabetical ordering for labels.   

We'll want to adjust this path to follow existing patterns and add a service owners section as well.

Reply:
Okay I've moved it and added the extra labels